src/app/main.cpp

```diff
@@ -376,6 +380,14 @@ void sigAbnormalHandler(int signum)
 #endif // defined(Q_OS_UNIX) || defined(STACKTRACE_WIN)
 
 #ifndef DISABLE_GUI
+void setupDpi()
+{
+#if defined(Q_OS_UNIX)
```

Comment:
If that is the case, we can still change the if-defs in the future. So please, change it and merge if nothing else is pending.